src/utils/feed.ts

```diff
@@ -8,7 +8,7 @@ export const createFeed = (team: PublicTeamResult, teamSlug: string) => {
   const date = new Date();
   const ogImage = generateTeamOGUrl(team?.slug || '');
   const feed = new Feed({
-    title: team!.name,
+    title: `Cool stuff from the Web - ${team!.name} - Digest.club`,
```

Comment:
`Cool stuff from the Web` should not be a hardcoded value, sadly we don't have a way to define a digest feed name in the app since a team can post anything.